src/pages/a/[platform]/[appId]/index.astro

```diff
@@ -21,12 +21,10 @@ const analyses = await e
     }))
     .run(client);
 
-if (analyses.length === 0) return new Response('We have not analyzed this app yet.', { status: 404 });
-
 // An app can theoretically change its name with each new version, we thus use the one we saw in the most recent proceeding.
-const name = analyses[0]!.proceeding.appName;
+const name = analyses[0]?.proceeding.appName || '<unknown app>';
```

Comment:
![image](https://github.com/user-attachments/assets/65899a7d-0020-43a3-ba87-d95714166877)
 I get this for every app.